### PR TITLE
Copter: 4.0 updates...part 1

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: copter/source/docs/failsafe-landing-page.rst [hunk 1]
```diff
@@ -20,4 +20,7 @@ The main failsafe topics are listed below.
     Polygon Fence <polygon_fence>
     Crash Check <crash_check>
     Parachute <parachute>
-    ADSB Avoidance of manned aircraft <common-ads-b-receiver>
\ No newline at end of file
+    ADSB Avoidance of manned aircraft <common-ads-b-receiver>
+    Hardware Watchdog <common-watchdog>
```
Comment (rmackay9) [inline copter/source/docs/failsafe-landing-page.rst]: Is it perhaps a "CPU watchdog" instead of "Hardware Watchdog"?

Comment (Hwurzburg) [reply copter/source/docs/failsafe-landing-page.rst]: Independent?

Comment (rmackay9) [reply copter/source/docs/failsafe-landing-page.rst]: let's call it the "Independent Watchdog" and maybe let's include this video from Marco? https://www.youtube.com/watch?v=ZGuTIPLI_e0

Comment (Hwurzburg) [reply copter/source/docs/failsafe-landing-page.rst]: done...

**Comment by Hwurzburg** [comment]
merging with review fixes

**Pull request merged**